### ISSUE TITLE
Example 8

### ISSUE THREAD

**Issue by rayzchen** [pyunity/pyunity#20]
Loading example 8 always gives an error:

```
Traceback (most recent call last):
  File "C:\Users\Ray\AppData\Local\Programs\Python\Python39-32\lib\runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\Ray\AppData\Local\Programs\Python\Python39-32\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "D:\Ray\python\pyunity\__main__.py", line 2, in <module>
    examples.show()
  File "D:\Ray\python\pyunity\examples\__init__.py", line 32, in show
    example.main()
  File "D:\Ray\python\pyunity\examples\example8\__init__.py", line 9, in main     
    source = scene.mainCamera.AddComponent(AudioSource)
  File "D:\Ray\python\pyunity\core.py", line 327, in AddComponent
    return self.gameObject.AddComponent(component)
  File "D:\Ray\python\pyunity\core.py", line 186, in AddComponent
    component = componentClass(self.transform)
  File "D:\Ray\python\pyunity\audio.py", line 58, in __init__
    mixer.Mix_AllocateChannels(channels)
OSError: exception: access violation reading 0x00000074
```

This doesnt happen when I run `python -m pyunity 8`, only when I run all examples. This happens on the `develop` branch.

**Comment by rayzchen** [comment]
The error only happens when a scene is loaded and unloaded before it, so it is something to do with closing the audio using `Mix_CloseAudio()`.

**Comment by rayzchen** [comment]
I think this is fixed with commit 425242c.
